### PR TITLE
EPLaunch classic to support two digit version numbers

### LINKED ISSUE #9186: EPLaunch can't handle upcoming version numbers
Issue overview
--------------
As we try to move to version 22.1, EPLaunch needs some tweaks to be able to handle it.  Basically the assumption is that the version is comprised of single digits.  The full details are here: https://github.com/NREL/EnergyPlus/pull/9169#issuecomment-966639236

### PR DESCRIPTION
**Pull request by JasonGlazer**
Pull request overview
---------------------
 - Fixes #9186 
 - Allows EP-Launch to handle version numbers in the form XX.X.X instead of just X.X.X

### Pull Request Author
Add to this list or remove from it as applicable.  This is a simple templated set of guidelines.
 - [x] Title of PR should be user-synopsis style (clearly understandable in a standalone changelog context)
 - [x] Label the PR with at least one of: Defect, Refactoring, NewFeature, Performance, and/or DoNoPublish
 - [ ] Pull requests that impact EnergyPlus code must also include unit tests to cover enhancement or defect repair
 - [ ] Author should provide a "walkthrough" of relevant code changes using a GitHub code review comment process
 - [ ] If any diffs are expected, author must demonstrate they are justified using plots and descriptions
 - [ ] If changes fix a defect, the fix should be demonstrated in plots and descriptions
 - [ ] If any defect files are updated to a more recent version, upload new versions here or on DevSupport
 - [ ] If IDD requires transition, transition source, rules, ExpandObjects, and IDFs must be updated, and add IDDChange label
 - [ ] If structural output changes, add to output rules file and add OutputChange label
 - [ ] If adding/removing any LaTeX docs or figures, update that document's CMakeLists file dependencies

### Reviewer
This will not be exhaustively relevant to every PR.
 - [ ] Perform a Code Review on GitHub
 - [ ] If branch is behind develop, merge develop and build locally to check for side effects of the merge
 - [ ] If defect, verify by running develop branch and reproducing defect, then running PR and reproducing fix
 - [ ] If feature, test running new feature, try creative ways to break it
 - [ ] CI status: all green or justified
 - [ ] Check that performance is not impacted (CI Linux results include performance check)
 - [ ] Run Unit Test(s) locally
 - [ ] Check any new function arguments for performance impacts
 - [ ] Verify IDF naming conventions and styles, memos and notes and defaults
 - [ ] If new idf included, locally check the err file and other outputs

### CODE REVIEW

File: src/EP-Launch/epl-ui.frm [hunk 1]
```diff
@@ -3242,10 +3242,16 @@ Dim extraRviNewName As String
...
 
 On Error Resume Next
 'create the file suffix for the old version
-oldVerSuffix = "_V" & Left(previousVersion, 1) & Mid(previousVersion, 3, 1) & Mid(previousVersion, 5, 1)
+firstPeriodPos = InStr(previousVersion, ".")
+If firstPeriodPos = 2 Then
+    oldVerSuffix = "_V" & Left(previousVersion, 1) & Mid(previousVersion, 3, 1) & Mid(previousVersion, 5, 1)
+Else
+    oldVerSuffix = "_V" & Left(previousVersion, 2) & Mid(previousVersion, 4, 1) & Mid(previousVersion, 6, 1)
+End If
```
Comment (Myoldmopar) [inline src/EP-Launch/epl-ui.frm:3254]:
```vb
Else if firstPeriodPos == 4 Then
    ' How are you still compiling VB6 in the year 2100!? 
```

File: src/EP-Launch/epl-ui.frm [hunk 2]
```diff
@@ -5676,12 +5695,13 @@ If transProg <> "" Then
       Mid(transProg, i) = "."
     End If
   Next i
+  toPos = InStr(transProg, "to")
+  exePos = InStr(transProg, "exe")
```
Comment (Myoldmopar) [inline src/EP-Launch/epl-ui.frm:5699]: This is exactly the type of change I was expecting.

**Review by Myoldmopar — APPROVED** [review]
From a code perspective alone, this looks great.  I would love it if @mjwitte or maybe @jcyuan2020 could go ahead and do the testing, but if I need to then I'll pull it onto the Windows machine and play around with it.

**Review by mjwitte — APPROVED** [review]
Tested with a package built from the latest version change branch:
1. Ran a v9.6 file - got a warning about an old idf before simulation started.
2. Transitioned v9.6 idf from EP-Launch File menu - transition utility ran.
3. Version number is correct now `Version,22.1;` and it runs without errors.
So, this fixes EP-Launch.

But IDFVersionUpdater also needs to be fixed. I added that to #9186

**Comment by Myoldmopar** [comment]
Awesome, thanks!  OK, let's merge this so I can get the version number change wrapped up and we can get some other PRs merged.

**Pull request merged**